shell/platform/darwin/macos/framework/Source/FlutterView.mm

```diff
@@ -107,7 +107,8 @@ - (void)didUpdateMouseCursor:(NSCursor*)cursor {
 // and will not restore it back.
 - (void)cursorUpdate:(NSEvent*)event {
   // Make sure to not override cursor when over a platform view.
-  NSView* hitTestView = [self hitTest:[self convertPoint:event.locationInWindow fromView:nil]];
+  NSPoint mouseLoc = [[self superview] convertPoint:event.locationInWindow fromView:nil];
```

Comment:
Can you use `mouseLocation` instead of `mouseLoc`?

Reply:
done